restcomm/restcomm.application/src/main/webapp/WEB-INF/scripts/mariadb/init.sql

```diff
@@ -188,7 +188,8 @@ voice_method VARCHAR(4),
 voice_fallback_url MEDIUMTEXT,
 voice_fallback_method VARCHAR(4),
 voice_application_sid VARCHAR(34),
-uri MEDIUMTEXT NOT NULL
+uri MEDIUMTEXT NOT NULL,
+push_client_identity VARCHAR(34)
```

Comment:
@agafox the `Push Client Identity` is the Client SID so there is no need to add an extra column to repeat the same data.